### PR TITLE
Add tests for ProperContains and ProperIn

### PR DESCRIPTION
**Pull request by antvaset**
Adding more tests for [ProperContains](https://cql.hl7.org/04-logicalspecification.html#proper-contains) and [ProperIn](https://cql.hl7.org/04-logicalspecification.html#proper-in) operators to cover more edge cases for the list overload.

Adjusting 4 existing tests to make it clear and make sure that ProperContainsNullRightFalse and ProperContainsNullRightTrue compile to ProperContains(List<T>, T) and that ProperInNullRightFalse and ProperInNullRightTrue compile to ProperIn(T, List<T>), matching the test name/group. Without null as String, the args could be resolved as List<T>, List<T> and compile to ProperIncludes(List<T>, List<T>) and ProperIncludedIn(List<T>, List<T>).

Unfortunately, the original PR (https://github.com/cqframework/cql-tests/pull/40) was closed automatically and cannot be reopened, so I have to open this new one.

Note that there's also an active thread under https://github.com/cqframework/clinical_quality_language/pull/1394.

### CODE REVIEW

**Review by raleigh-g-thompson — APPROVED** [review]

File: tests/cql/CqlListOperatorsTest.xml [hunk 1]
```diff
@@ -846,16 +846,66 @@
 	</group>
 	<group name="ProperContains" version="1.0">
 		<capability code="list-operators" />
+		<test name="ProperContains1" version="1.0">
+			<capability code="list-operators" />
+			<expression>null as List&lt;String&gt; properly includes 'a'</expression>
```
Comment (brynrhodes) [inline tests/cql/CqlListOperatorsTest.xml]:
```suggestion
			<expression>null as List&lt;String&gt; properly includes { 'a' }</expression>
```
